main.md

```diff
@@ -302,11 +304,28 @@ The client uses the `request_uri` value returned by the authorization server to
   request_uri=urn%3Aexample%3Abwc4JK-ESC0w8acc191e-Y1LTC2 HTTP/1.1
 ```
 
-# Authorization Server Metadata
+Authorization server policy MAY dictate that pushed authorization requests are the only means for clients to pass authorization request data. In this case, the authorization server will refuse, using the `invalid_request` error code, to process any request to the authorization endpoint that does not have a `request_uri` parameter with a value obtained from the pushed authorization request endpoint.
+
+The same error code is returned if the client's policy requires use of pushed authorization requests.
```

Comment:
It took me a couple of reads to understand that this is if it is a per-client policy rather than a blanket policy.,. But I can't immediately think of any better wording

Reply:
I took the wording from Torsten's similar PR but had similar reaction to it. I couldn't immediately think of any better wording either. But I'll give it another effort.

Reply:
82128dce36d46ce2af593e8214e4a38c735e549b hopefully makes this bit a little easier to read/understand